docs/presentation/Content/Terminology.md

```diff
@@ -44,47 +50,36 @@ Properties are considered similar if:
-| **Result** | **has value** | **\*\*\*Varies\*\*\*** | **\*\*\*Varies\*\*\*** |
-
-The above example would render into a similar property grid:
-
-| Field       | Value              |
-| ----------- | ------------------ |
-| **Field 1** | has value          |
-| **Field 2** | \*\*\*Varies\*\*\* |
-| **Field 3** | \*\*\*Varies\*\*\* |
+When content is requested with [ContentFlags.MergeResults]($presentation-common) flag and contains more than 1 result row, its values are merged into 1 record. This is generally used only for property pane use case where we may request content for multiple selected elements.
+
+> **Example** of merged values:
+>
+> | Row        | Field 1       | Field 2                | Field 1                |
+> | ---------- | ------------- | ---------------------- | ---------------------- |
+> | row 1      | has value     | has value              | -                      |
+> | row 2      | has value     | -                      | has value              |
+> | **Result** | **has value** | **\*\*\*Varies\*\*\*** | **\*\*\*Varies\*\*\*** |
+>
+> The above example would render into a similar property grid:
+>
+> | Field       | Value              |
+> | ----------- | ------------------ |
+> | **Field 1** | has value          |
+> | **Field 2** | \*\*\*Varies\*\*\* |
+> | **Field 3** | \*\*\*Varies\*\*\* |
 
 ## Nested Content
 
-In most cases property values are primitive, arrays or structs. However,
-it is possible to set up [related properties](#related-properties) using
-one-to-many or many-to-many relationship which creates, what we call,
-nested content.
-
-Example:
-
-```text
-           Model
-          /  |  \
-         /   |   \
-        /    |    \
-Element1 Element2 Element3
-```
-
-In the above example *Model* has 3 related *Element* instances. We can
-request content for *Model* and ask to additionally show related *Element*
-properties. In this case element properties are be called
-**nested content** because properties of multiple *Element* instances are
-displayed as a single *Model* property. The property is created as an array
-of structs where each struct is created from *Element* instance.
+In most cases property values are primitive, arrays or structs. However, it is possible to set up [related properties](#related-properties) using one-to-many or many-to-many relationship which creates, what we call, the nested content.
+
+> **Example:**
+>
+> ```text
+>           Model
+>           /  |  \
+>         /   |   \
+>         /    |    \
+> Element1 Element2 Element3
+> ```
+>
+> In the above example *Model* has 3 related *Element* instances. We can request content for *Model* and ask to additionally > show related *Element* properties. In this case element properties are be called **nested content** because properties of > multiple *Element* instances are displayed as a single *Model* property. The property is created as an array of structs where each struct is created from *Element* instance.
```

Comment:
```suggestion
> In the above example *Model* has 3 related *Element* instances. We can request content for *Model* and ask to additionally show related *Element* properties. In this case element properties are called **nested content** because properties of multiple *Element* instances are displayed as a single *Model* property. The property is created as an array of structs where each struct is created from *Element* instance.
```